build_src.cmake

```diff
@@ -37,6 +37,7 @@ set(LIBXMP_SRC_LIST_LITE
 # Sources included into the full only version
 set(LIBXMP_SRC_LIST_FULL
     src/extras.c
+    src/filetype.c
```

Comment:
The lite version needs this source, too.

Reply:
Whoops, missed this up, damn